test/Kestrel.Transport.FunctionalTests/ChromeTests.cs

```diff
@@ -40,33 +42,33 @@ public class ChromeTests : LoggedTest
         private string NetLogPath { get; set; }
         private string StartupLogPath { get; set; }
         private string ShutdownLogPath { get; set; }
-        private string ChromeArgs { get; set; }
+        private string[] ChromeArgs { get; set; }
 
         private void InitializeArgs()
         {
             NetLogPath = Path.Combine(ResolvedLogOutputDirectory, $"{ResolvedTestMethodName}.nl.json");
             StartupLogPath = Path.Combine(ResolvedLogOutputDirectory, $"{ResolvedTestMethodName}.su.json");
             ShutdownLogPath = Path.Combine(ResolvedLogOutputDirectory, $"{ResolvedTestMethodName}.sd.json");
 
-            ChromeArgs = $"--headless " +
-                $"--no-sandbox " +
-                $"--disable-gpu " +
-                $"--allow-insecure-localhost " +
-                $"--ignore-certificate-errors --enable-features=NetworkService " +
-                $"--enable-logging " +
-                $"--dump-dom " +
-                $"--virtual-time-budget=10000 " +
-                $"--log-net-log={NetLogPath} " +
-                $"--trace-startup --trace-startup-file={StartupLogPath} " +
-                $"--trace-shutdown --trace-shutdown-file={ShutdownLogPath}";
+            ChromeArgs = new [] {
+                $"--headless ",
+                $"--no-sandbox ",
+                $"--disable-gpu ",
+                $"--allow-insecure-localhost ",
+                $"--ignore-certificate-errors --enable-features=NetworkService ",
```

Comment:
Why are you including multiple argument flags in a single array element?

Reply:
Good point, I didn't notice when I was updating the format.